modules/eslint-plugin/src/rules/index.ts

```diff
@@ -71,4 +73,7 @@ export const rules = {
   'use-consistent-global-store-name': useConsistentGlobalStoreName,
   // operators
   'prefer-concat-latest-from': preferConcatLatestFrom,
+  // signals
+  'signal-state-no-arrays-at-root-level': signalStateNoArraysAtRootLevel,
```

Comment:
The actual fix to include the signal rules in ESLint v8.
I added extra tests to catch this in the future.